docs/specs/restore.yml

```diff
@@ -163,25 +163,6 @@ outputs:
       "conceptual": "<p>for long file path test.</p>"
     }
 ---
-# [Skip] Restore private azure repos with dependencies
-# VSTS recently are not really stable, so skip this test for now
```

Comment:
Can we enable it after this PR is merged?

Reply:
enabling that requires modifying test framework to support base64 encoding (expressions?), since this scenarios is main for integration, it is already covered by integration tests, so I don't plan to enable it again.